modules/analysis/src/service/mod.rs

```diff
@@ -129,20 +129,48 @@ impl AnalysisService {
     ///
     /// A new instance will have a new cache. Instanced cloned from it, will share that cache.
     ///
+    /// Default cache size is 100MB.
+    ///
     /// Therefore, it is ok to create a new instance. However, if you want to make use of the
     /// caching, it is necessary to re-use that instance.
     ///
     /// Also, we do not implement default because of this. As a new instance has the implication
     /// of having its own cache. So creating a new instance should be a deliberate choice.
     #[allow(clippy::new_without_default)]
     pub fn new() -> Self {
+        Self::new_sized(1024 * 1024 * 100)
```

Comment:
I am worries this gets overlooked. It looks like we use this for tests only. One way could be to add `#[cfg(test]`. Or drop it an have a `const` default value for tests.

Reply:
I'll just drop it and fix it in the tests.